src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.Process;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
+
+/** Servlet that actions items.*/
+@WebServlet("/actions")
+public class ActionsServlet extends HttpServlet {
+
+  private static final List<String> terms = Arrays.asList("Black Lives Matter", "COVID-19");
+  private static final String API_KEY = "API_KEY";  // Insert the API_KEY here for testing.
+  private static final String API_PATH = "https://api.globalgiving.org/api/public/services/search/projects";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> jsonResultList = new ArrayList<>();
+    for (String term : terms) {
+      String queryTerm = encodeTerm(term);
+      String jsonResult = curlProjects(API_KEY, queryTerm);
+      jsonResultList.add(jsonResult);
+    }
+    String jsonBase = "{\"results\": [%s]}";
```

Comment:
[Nit] Since you only use this variable once it is probably better to just hardcode.